code/modules/clothing/suits/jobs.dm

```diff
@@ -68,38 +20,31 @@ obj/item/clothing/suit/apron/overalls/blue
 	flags_inv = HIDESHOES|HIDEJUMPSUIT
 	allowed = list(/obj/item/tank/emergency_oxygen,/obj/item/storage/bible,/obj/item/nullrod,/obj/item/reagent_containers/food/drinks/bottle/holywater)
 
-//Chef
-/obj/item/clothing/suit/chef
-	name = "chef's apron"
-	desc = "An apron used by a high class chef."
+/********** Chef/Cook Start **********/
+// Chef Jacket
+/obj/item/clothing/suit/chef_jacket
```

Comment:
Revert chef_jacket to just chef. Will just cause confusion with jacket and coat items.

If you really want to, use *uniform* instead, as it is the **actual** slot it resides in.

Same with all the other names and descriptions, change them as well. It's not a jacket, because you can't wear them OVER anything.

Reply:
it is a jacket.

Reply:
...no? its for things such as /obj/item/clothing/suit/storage/toggle/

Reply:
still a jacket, toggle-able or not is out of scope for this PR.

Reply:
if it isn't toggleable, then don't call it a jacket. just revert the name and i'll approve the pr.